assets/scripts/fe/src/app/components/users/setting/setting.component.ts

```diff
@@ -18,6 +25,12 @@ import { NavService } from '../../../commons/services/utils/nav.service';
 export class SettingComponent implements OnInit {
   private form : UserForm;
 
+  private edit_password_form : EditPasswordForm
+  private add_password_form : AddPasswordForm
+  checkPass;
```

Comment:
always define the member of a variable. never leave the variable unsure of the type. Also add a type and default value, atleast a `type`.

Reply:
member is `public` or `private
type is `data type (string, int, class,...)`